skiko/src/linuxMain/kotlin/org/jetbrains/skiko/SkiaLayer.linux.kt

```diff
@@ -93,6 +93,7 @@ actual enum class SkikoKey(val value: Int) {
     KEY_RIGHT_SHIFT(60),
```

Comment:
May make sense to move to separate file as well.

Reply:
Done.